src/components/Table/Table.tsx

```diff
@@ -38,7 +122,10 @@ export const Table = <T extends object>({ headers, data, handleClickRow, stylesC
             <tr style={stylesColumn} key={index} onClick={(event) => handleClickRow && handleClickRow(event, datum)}>
               {Object.keys(headers).map((key, index) => {
                 return (
-                  <td key={index} style={{ width: headers[key].width, textAlign: headers[key].align, padding: headers[key].padding }}>
+                  <td
+                    key={index}
+                    style={{ width: headers[key].width, textAlign: headers[key].align, padding: headers[key].padding }}
```

Comment:
can we use class?  If inline is a must to make it effective, let's use variable.